src/lib/components/HarnessSelector/HarnessSelector.svelte

```diff
@@ -28,17 +28,21 @@
   $: browser && $harnesses.length > 0, setInitialSelection();
   $: {
     onChange(selection);
-    if (selection) {
-      updateQueryParams(selection);
-    }
+    updateQueryParams(selection);
   }
 
   function updateQueryParams(selectedHarness) {
-    const [make, ...model] = selectedHarness.car.split(' ');
     const searchParams = new URLSearchParams();
-    searchParams.set("make", encodeURIComponent(make));
-    if (model.length > 0) searchParams.set("model", encodeURIComponent(model.join(' ')));
-    goto(`?${searchParams.toString()}`, { keepfocus: true, replaceState: true, noScroll: true });
+    if (selectedHarness) {
+      const [make, ...model] = selectedHarness.car.split(' ');
+      searchParams.set("make", encodeURIComponent(make));
+      if (model.length > 0) searchParams.set("model", encodeURIComponent(model.join(' ')));
```

Comment:
I wonder if make is ever set without model... @incognitojam can we just set `name` instead?

Reply:
Yes, it happens for the generic/developer harnesses that are a single word (e.g. "GM", or "Developer") since it's splitting on the car name, which is actually a harness in some cases.

Reply:
Yeah I can't think of a reason knowing the make without the model is useful anyway